ray-operator/test/sampleyaml/support.go

```diff
@@ -101,3 +101,30 @@ func SubmitJobsToAllPods(t Test, rayCluster *rayv1.RayCluster) func(Gomega) {
 		}
 	}
 }
+
+func GetApps(rayService *rayv1.RayService) map[string]rayv1.AppStatus {
+	apps := make(map[string]rayv1.AppStatus)
+	for k, v := range rayService.Status.PendingServiceStatus.Applications {
```

Comment:
Don't need to check `PendingServiceStatus`. `PendingServiceStatus` will transition to "active" when all serve applications are running and ready.